src/scripts/model/game.js

```diff
@@ -0,0 +1,93 @@
+import Round from "./round";
+import board from '../board';
+
+export default class Game {
+
+    /*
+    * @param teams = []
+    * @param questions QuestionStore
+    */
+    constructor(teams, questionsStore) {
+        this.teams = teams;
+        this.questionsStore = questionsStore;
+        this.roundCount = 1;
+        this.round = new Round(this.questionsStore.getRandomQuestion());
+        this.setRandomTeam();
+    }
+
+    startNewRound() {
+        this.switchCurrentTeam();
+        this.resetTeamsErrors();
+        this.roundCount++;
+        this.round = new Round(this.questionsStore.getRandomQuestion());
+    }
+
+    resetTeamsErrors() {
+        for(let team of this.teams) { team.resetErrors() }
+    }
+
+    /**
+     * @param {string} playerAnswer 
+     */
+    handlePlayerAnswer(playerAnswer) {
+        const result = this.resolvePlayerAnswer(playerAnswer);
+        const currentTeam = this.getCurrentTeam();
+
+        switch(result.status) { 
+            case true: { 
+               this.round.setBoardAnswer(result.answer, this);
+               break; 
+            } 
+            case false: { 
+               this.round.setBoardError(currentTeam, this);
+               break; 
+            }
+            default: { 
+               throw new Error('Result have only true of false status');
+            } 
+        }
+    }
+    /**
+     * @param {string} playerAnswer 
+     */
+    resolvePlayerAnswer(playerAnswer) {
+
+        const answer = this.round.getQuestion().getAnswers().find(answer => {
+            return answer.ans.toLowerCase() === board.removeDiacritics(playerAnswer);
+        });
+
+        return (answer !== undefined) ? { answer, status: true} : { answer: null, status: false};
+    }
+
+    getQuestionsStore() {
+        return this.questionsStore;
+    }
+
+    switchCurrentTeam() {
+        this.setCurrentTeam(this.teams.find(team => team.getName() !== this.currentTeam).name)
+    }
+
+    setRandomTeam() {
+        this.setCurrentTeam(this.teams[Math.round(Math.random())].name);
+    }
+
+    setCurrentTeam(teamName) {
+        this.currentTeam = teamName;
```

Comment:
this function makes another action - this function doesn't set team, but sets name of team